portal-impl/src/custom-sql/documentlibrary.xml

```diff
@@ -138,7 +138,7 @@
 	<sql id="com.liferay.document.library.kernel.service.persistence.DLFileEntryFinder.findByExtraSettings">
 		<![CDATA[
 			SELECT
-				DISTINCT {DLFileEntry.*}
+				DISTINCT DLFileEntry.fileEntryId, DLFileEntry.modifiedDate
```

Comment:
Why are we including the modified date? Is it necessary for the order by clause?

Reply:
Yes, if we don't include modifiedDate, then the query will error out due to the modifiedDate being used in the order by clause. In Oracle, the error is "ORA-01791: not a SELECTed expression".